src/main/kotlin/zhi/yest/vk/friendfinder/domain/Group.kt

```diff
@@ -0,0 +1,54 @@
+package zhi.yest.vk.friendfinder.domain
+
+import com.fasterxml.jackson.annotation.JsonAlias
+import com.fasterxml.jackson.annotation.JsonProperty
+
+data class Group(val id: Int,
+                 @JsonAlias("name")
+                 @JsonProperty("name")
+                 val name: String,
+                 @JsonAlias("screen_name")
+                 @JsonProperty("screen_name")
+                 val screenName: String,
+                 @JsonAlias("is_closed")
+                 @JsonProperty("is_closed")
+                 val closed: Boolean,
+                 @JsonAlias("type")
+                 @JsonProperty("type")
+                 val type: String,
+                 @JsonAlias("is_admin")
+                 @JsonProperty("is_admin")
+                 val admin: Boolean,
+                 @JsonAlias("admin_level")
```

Comment:
why do we need alias, when we have property? WIP - need to return camel cased fields